src/sync/events/utils.ts

```diff
@@ -64,11 +73,23 @@ export const fetchBlock = async (chainId: number, blockNumber: number, force = f
                 gasPrice,
                 gasUsed,
                 gasFee,
+                gasLimit,
+                nonce: 1,
```

Comment:
Please don't hardcode value here

Reply:
Was a typo. Changing it to `nonce: tx.nonce`